lib/src/user.dart

```diff
@@ -172,7 +172,8 @@ class User extends Event {
   /// Whether the client is allowed to change the power level of this user.
   /// Please be aware that you can only set the power level to at least your own!
   bool get canChangePowerLevel =>
-      room.canChangePowerLevel && powerLevel < room.ownPowerLevel;
+      (room.canChangePowerLevel && powerLevel < room.ownPowerLevel) ||
+      (id == room.client.userID && powerLevel > 0);
```

Comment:
do you actually need `powerLevel > 0`?

Reply:
Hmm in the case this are you self, you can change your own powerlevel only in one direction. So I assumed this makes no sense to return true if the powerlevel is 0

Reply:
yes but you can always create lower power levels, the spec allows you `[-(2**53)+1, (2**53)-1]`

Reply:
okay I checked it out. The correct calculation should be that you:
1. Have the permission to change power levels in general
2. Your power-level is higher than the power level of the target user OR target user is yourself!